rust+wasm/{{project-name}}-wasm/README.md

```diff
@@ -94,7 +94,7 @@ Javascript-wrapper file, the {{project-name}}-wasm README (and version), and a
 For running tests in the current directly, use this command:
```

Comment:
Not actually in the current PR, but noticed a possible typo here. Should this be "current directory"?